packages/front-end/components/Experiment/MetricsOverridesSelector.tsx

```diff
@@ -209,8 +207,8 @@ export default function MetricsOverridesSelector({
                           placeholder="default"
                           helpText={
                             <div className="text-right">
-                              {/* @ts-expect-error TS(2532) If you come across this, please fix it!: Object is possibly 'undefined'. */}
-                              default: {metricDefinition.conversionWindowHours}{" "}
+                              default:{" "}
+                              {metricDefinition?.conversionWindowHours || ""}{" "}
```

Comment:
should probably be `??` in case hours are legitimately 0

Reply:
Good call. I made the change here 1fe0ccdc511a3311fae1845311abc63c9e9fcb7a